sklearn/tests/test_kernel_ridge.py

```diff
@@ -19,7 +18,9 @@ def test_kernel_ridge():
     assert_array_almost_equal(pred, pred2)
 
 
-def test_kernel_ridge_csr():
+@pytest.mark.parametrize("csr_container", CSR_CONTAINERS)
```

Comment:
Maybe these two tests can be combined since they look completely identical?

```python
@pytest.mark.parametrize("sparse_container", [*CSC_CONTAINERS, *CSR_CONTAINERS])
def test_kernel_ridge_sparse(sparse_container):
    X_sparse = sparse_container(X)
    ...
```

Reply:
@kianelbo emm, I think you would need [*CSC_CONTAINERS, *CSR_CONTAINERS] to unpack them, or equivalently CSC_CONTAINERS + CSC_CONTAINERS.